src/main/java/org/jitsi/jigasi/stats/Statistics.java

```diff
@@ -209,6 +234,41 @@ public class Statistics
             TOTAL_CALLS_NO_HEARTBEAT,
             "Total number of calls dropped due to no response to sip heartbeat.");
 
+    /**
+     * Total number of transcriptions started.
+     */
+    private static CounterMetric totalTrasnscriberStarted = JigasiMetricsContainer.INSTANCE.registerCounter(
+            TOTAL_TRANSCRIBER_STARTED,
+            "Total number of started transcriptions.");
+
+    /**
+     * Total number of transcriptions stopped.
+     */
+    private static CounterMetric totalTrasnscriberStopped = JigasiMetricsContainer.INSTANCE.registerCounter(
+            TOTAL_TRANSCRIBER_STOPPED,
+            "Total number of stopped transcriptions.");
+
+    /**
+     * Total number of transcriptions failes.
+     */
+    private static CounterMetric totalTrasnscriberFailed = JigasiMetricsContainer.INSTANCE.registerCounter(
+            TOTAL_TRANSCRIBER_FAILED,
+            "Total number of failed transcriptions.");
+
+    /**
+     * The total number of 15 second intervals submitted to the Google API for transcription.
+     */
+    private static LongGaugeMetric total15sIntervals = JigasiMetricsContainer.INSTANCE.registerLongGauge(
```

Comment:
Why the weird unit of "15 seconds"? Can't we just use seconds or minutes?

Reply:
Done.